hooks/post/10-post_install_message.rb

```diff
@@ -1,30 +1,28 @@
 # Puppet status codes say 0 for unchanged, 2 for changed succesfully
 if [0,2].include? @kafo.exit_code
-  say "  <%= color('Success!', :good) %>"
+  Kafo::MessageHelpers.success_message
 else
-  say "  <%= color('Something went wrong!', :bad) %> Check the log for ERROR-level output"
+  Kafo::MessageHelpers.failure_message
 end
 
 # Foreman UI?
 if module_enabled? 'foreman'
-  say "  * <%= color('Foreman', :info) %> is running at <%= color('#{param('foreman','foreman_url').value}', :info) %>"
-  say "      Initial credentials are <%= color('#{param('foreman', 'initial_admin_username').value}', :info) %> / <%= color('#{param('foreman', 'initial_admin_password').value}', :info) %>"
+  Kafo::MessageHelpers.server_success_message(@kafo)
+  Kafo::MessageHelpers.new_install_message(@kafo) if new_install?
 end
 
-# Proxy?
-if module_enabled? 'foreman_proxy'
-  proxy_url = param('foreman_proxy','registered_proxy_url').value
-  if proxy_url.nil? || proxy_url.empty?
-    proxy_url = "https://#{param('foreman_proxy','registered_name').value}:#{param('foreman_proxy','ssl_port').value}"
-  end
-  say "  * <%= color('Foreman Proxy', :info) %> is running at <%= color('#{proxy_url}', :info) %>"
+if module_enabled?('katello')
+  Kafo::MessageHelpers.certs_generate_command_message
+end
+
+if module_enabled?('katello_devel')
+  Kafo::MessageHelpers.dev_server_success_message(@kafo)
+  Kafo::MessageHelpers.dev_new_install_message(@kafo) if new_install?
 end
 
-# Puppetmaster?
```

Comment:
Any reason to drop this?

Reply:
Do we announce any other service that is running? Seemed odd to me. We are installing Foreman, that should be all we point users to and they can go figure out where things are running if needed from the smart proxy attached.